fastlane/lib/fastlane/actions/jira.rb

```diff
@@ -90,6 +95,14 @@ def self.example_code
             password: "Your password",
             ticket_id: "Ticket ID, i.e. IOS-123",
             comment_text: "Text to post as a comment"
+          )',
+          'jira(
+            url: "https://yourserverdomain.com",
+            context_path: "/jira", # append \'/jira\' path to your url
```

Comment:
Can you remove the `# append \'/jira\' path to your url`? This information should be in the option description 😊